src/plugins/facebook/facebook.js

```diff
@@ -0,0 +1,58 @@
+/**
+* @title WET-BOEW Facebook embedded page
+* @overview Helps with implementing Facebook embedded pages.
+* @license wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+* @author @pjackson28
+*/
+( function( $, window, wb ) {
+	"use strict";
+
+	/*
+	* Variable and function definitions.
+	* These are global to the plugin - meaning that they will be initialized once per page,
+	* not once per instance of plugin on the page. So, this is a good place to define
+	* variables that are common to all instances of the plugin on a page.
+	*/
+	var componentName = "wb-facebook",
+	selector = "." + componentName,
+	initEvent = "wb-init" + selector,
+	$document = wb.doc,
+	fbinited = false,
+
+	/**
+	* @method init
+	* @param {jQuery Event} event Event that triggered the function call
+	*/
+	init = function( event ) {
+
+		// Start initialization
+		// returns DOM object = proceed with init
+		// returns undefined = do not proceed with init (e.g., already initialized)
+		var ele = wb.init( event, componentName, selector );
+
+		if ( ele ) {
+			Modernizr.load(
+				{
+					load: [ "//connect.facebook.net/" + wb.lang + "_CA/sdk.js" ],
+					complete: function() {
+						if ( !fbinited ) {
```

Comment:
I was just answering your questions regarding <code>wb-init.wb-twitter</code>. The <code>fbinited</code> approach is consistent with the i18n approach that I was referring to.